Sprint 2/UserDAO.java

```diff
@@ -0,0 +1,55 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+public class UserDAO {
+	private Connection connect = null;
+	private Statement statement = null;
+	private PreparedStatement preparedStatement = null;
+	private ResultSet resultSet = null;
+	private DBConnection dbConnection = null;
+	
+	private void createTable() throws SQLException {

```

Comment:
This method would be a lot easier to read if you indent the lines of code properly.